gql_example_flutter/test/widget_test.dart

```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
```

Comment:
This file should probably be dropped.

Reply:
complete in https://github.com/gql-dart/gql/pull/33/commits/753e3957e9a633af214c219a61918289341ce901